CONTRIBUTING.rst

```diff
@@ -3,24 +3,20 @@ Contributing guide
 
 Thanks for your interest in contributing to cpplint.
 
-Any kinds of contributions are welcome: Bug reports, Documentation, Patches.
+Any kinds of contributions are welcome: Bug reports, Documentation, Patches. If you need some ideas, you may check out some of the tasks below:
 
-However, cpplint is a bit special as a project because it aims to closely follow what Google does in the upstream repository.
-That means Google remains the source of all major requirements and functionality of cpplint whereas this fork adds extensions to cpplint run on more environments and in more companies.
-The difference between this cpplint and Google should remain so small that anyone can at a glance see there is no added change that could be regarded as a security vulnerability.
+* Reducing interference with other popular tools
+* Update cpplint for modern python versions
```

Comment:
this has been solved

Reply:
Future modern python versions can still break cpplint

Reply:
When the project is being maintained, there will be issues created for that very quickly, and they will be fixed quickly.  No need to ask newbies to be attempting that type of task.